src/Components/Blueprints/ImportBlueprintModal.tsx

```diff
@@ -96,15 +102,25 @@ export const ImportBlueprintModal: React.FunctionComponent<
             <HelperText>
               <HelperTextItem variant={isRejected ? 'error' : 'default'}>
                 {isRejected
-                  ? 'Must be a JSON file no larger than 1 KB'
+                  ? 'Must be a JSON file no larger than 6 KB'
```

Comment:
In case we want to export even the gpg key and overall everything, 1 KB does not seem to be enough.